docs/dg/dev/integrate-and-configure/integrate-sitemap-generation.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Integrate Sitemap module
+description: Learn the prerequisites and how to enable and integrate Sitemap module into a Spryker based project.
+last_updated: March 4, 2025
+template: howto-guide-template
+---
+
+The Sitemap module helps in generating sitemaps for your Spryker application, which can improve the SEO of your site by making it easier for search engines to index your pages.
+The Sitemap module provides various configuration options to customize the sitemap generation process according to your specific needs.
+This document describes how to integrate Sitemap module into a Spryker project.
+
+### Prerequisites
+
+To start the integration, install the necessary features:
+
+| NAME                  | VERSION          | INSTALLATION GUIDE                                                                                                                                               |
+|-----------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core          | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{site.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html)     |
+
+## 1) Install required modules using Composer
+
+```bash
+composer require spryker/category-storage:"^2.10.0" spryker/cms-storage:"^2.8.0" spryker/merchant-storage:"^1.3.0" spryker/product-set-storage:"^1.11.0" spryker/product-storage:"^1.42.0" spryker/sitemap:"^0.1.0" spryker/sitemap-extension:"^1.0.0" spryker-shop/shop-ui:"^1.85.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE           | EXPECTED DIRECTORY                 |
+|------------------|------------------------------------|
+| CategoryStorage  | vendor/spryker/category-storage    |
+| CmsStorage       | vendor/spryker/cms-storage         |
+| MerchantStorage  | vendor/spryker/merchant-storage    |
+| ProductSetStorage| vendor/spryker/product-set-storage |
+| ProductStorage   | vendor/spryker/product-storage     |
+| Sitemap          | vendor/spryker/sitemap             |
+| SitemapExtension | vendor/spryker/sitemap-extension   |
+| ShopUi           | vendor/spryker-shop/shop-ui        |
+
+{% endinfo_block %}
+
+## 2) Adjust configuration 
+
+### 2.1) Configure the filesystem service for Sitemap: 
+
+<details>
+<summary>config/Shared/config_default.php</summary>
+
+```php
+use Spryker\Shared\Sitemap\SitemapConstants;
+
+$config[FileSystemConstants::FILESYSTEM_SERVICE] = [
+    SitemapConstants::FILESYSTEM_NAME => [
```

Comment:
please add some comments here that the first one for syncing file between containers and the second one for the local cache. Please also add a warning that files can be quite big and projects should take it in consideration

Reply:
or mb leave a link on your cache section below